rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -186,6 +177,29 @@ public static ISchedulerClient createInstance() {
                                                          new SessionHandler(client));
     }
 
+    private static String createJobSortParamsString(List<SortParameter<JobSortParameter>> jobSortParameters) {
+
+        String jobSortParametersAsString = "";
```

Comment:
Please use a StringBuilder instead of concatenating in the loop

Reply:
No more used in the next commit.